### PR TITLE
Feature/modal functionality

### PR DESCRIPTION
**Pull request by rdtho2525**
# Selector FE PR

## What does this merge do? Check all that apply.
- [ ] Adds a new feature
- [X] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## Summary of changes
- Queries for Spotify ID
- Applies React Router to navigate to the details page
- Converts origin modal design to a standard page layout

## Notes for reviewer (any known bugs, areas of interest, etc.)
- AlbumCards were made into Links in this branch, however, a better solution has been provided by @Max9545; defer to his branch changes when merge conflicts arise

## Next Steps (Future Iterations)
- TBD

### CODE REVIEW

File: src/queries.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
 import { gql } from '@apollo/client';
 
 export const GET_SINGLE_ALBUM = gql`
+  query ($title: String!) {
```
Comment (Max9545) [inline src/queries.js:4]: would love to know why this wasnt working

**Comment by Max9545** [comment]
Lighthouse at 100%! nice!

**Comment by Max9545** [comment]
A you tube video would fill in the bottom left corner quite nicely

File: src/queries.js (hunk 1 above)
Comment (rdtho2525) [reply src/queries.js:4]: Perhaps it was just early in the process. Thankful it works now!

**Pull request merged**